addon/components/upf-stat.js

```diff
@@ -15,5 +15,11 @@ export default Component.extend({
   label: null,
 
   icon: null,
-  iconClass: ''
+  iconClass: '',
+  iconUrl: null,
+  iconLabel: null,
+
+  didRender() {
+    this.$('[title]').tooltip({ placement: 'bottom' });
```

Comment:
may be wrap it in: `if (!isBlank(this.get('iconLabel')) {`

Reply:
Great idea :) thanks